<?xml version="1.0"?>
<launch>
  <param name="points_topic" value="/camera_lidar_fusion/seg_cloud" />
  <param name="global_map_downsample_resolution" value="0.1" />
  <param name="output_inlier_fraction_threshold" value="0.9" />
  <param name="global_map_path" value="$(find hdl_localization)/data/6_new_mesh.pcd" />
  <!-- BBS FPFH_RANSAC -->
  <param name="engine_name" value="FPFH_RANSAC" />
  <node name="hdl_global_localization_test" pkg="hdl_global_localization" type="hdl_global_localization_test" output="screen"/>
  <node name = "view_global_localization" pkg="rviz" type="rviz" args="-d $(find hdl_global_localization)/rviz/view_global_localization.rviz"/>
</launch>
